src/components/AccordionPanel/AccordionPanel.module.css

```diff
@@ -171,20 +171,9 @@
  */
 .accordion-panel__icon {
   margin-left: auto;
-  height: var(--eds-size-4);
-  width: var(--eds-size-4);
-  fill: var(--eds-theme-color-primary-foreground);
```

Comment:
this token no longer exists so I replaced it with `icon-neutral-strong` throughout 🤷‍♀️  sure we'll polish it later